Revisions for resubmission (#5)
rando2/dtcdog-ms: content/04.results.md

All other breeds are represented in other colors as identified in the legend.
](images/test_results.png){#fig:pies secno=1}

> ### MAPPING TO STANDARDIZED TERMINOLOGIES ENABLES COMPARISONS

Reviewer: ```suggestion
### Mapping to Standardized Terminologies Enables Cross-test Comparisons
```

> In most cases, test results could be mapped to the VBO [@doi:10.5281/zenodo.7996675; @icbo-vbo].

Reviewer: ```suggestion
Test results were usually directly mapped to VBO terms [@doi:10.5281/zenodo.7996675; @icbo-vbo].
```

> Only four breed estimates mapped ambiguously to terms in the VBO (Supplementary Table 1).

Reviewer: ```suggestion
However, four breed estimates mapped ambiguously to terms in the VBO (Supplementary Table 1).
```

> In one case, a result returned by NASNet ("Lhasa") is not a known dog breed, so "Lhasa Apso" was assumed to be the intended breed.

Reviewer: ```suggestion
In one case, the result returned by NASNet ("Lhasa") was not a known dog breed, so "Lhasa Apso" was assumed to be the intended breed.
```

In the other three cases, two or more potential VBO terms could correspond to the breed result.

> In two cases, VBO developers were consulted to select the best mapping.

Reviewer: ```suggestion
For two cases, VBO developers were consulted to select the best mapping.
```

The last ambiguous breed estimate was Catahoula Leopard Dog (DNA My Dog, estimated for German Short-Haired Pointer DNA submitted with Italian Greyhound photo).
The Catahoula Leopard Dog is a breed with a complex history that could correspond to three VBO entries.

> However, as this result was returned only by one test for one dog, it did not need to be standardized across tests, so we simply selected the closest VBO term to the exact phrasing used in the breed result.

Reviewer: ```suggestion
However, as this result was returned only by one test for one dog, we selected the closest VBO term to the exact phrasing used in the breed result.
```

Also, due to sample loss, only 11 dogs were tested using Darwin's Ark and 9 for Accu-Metrics versus 12 for the other tests.
{#tbl:accuracy}

> ### AI-BASED PREDICTION WITH CAININE PHOTOGRAPHS WAS ACCURATE WITH INCLUDED BREEDS

Reviewer: ```suggestion
### AI-Based Prediction with Canine Photographs was Accurate with Breeds Included in Training
```

The neural network NASNet was, as expected, unable to predict DNA donor breed in the treatment condition, where the photograph and DNA sample did not match (Figure @fig:pies).
While the neural network was often able to correctly identify the registered breed of the dog in the photographs it was provided, it sometimes identified breeds that were neither the DNA donor nor the photographed dog.
This pattern was likely influenced by the limited number of breeds in the training data, and thus category labels, used by ImageNet [@url:http://vision.stanford.edu/aditya86/ImageNetDogs].
For example, ImageNet does not include the Chinese Crested and Bulldog breeds, and therefore NASNet also failed to identify these breeds (Figure @fig:pies; Supplementary Table 4).

> ### MOST TESTS ACCURATELY REFLECTED PUREBRED DOGS’ REGISTERED BREEDS

Reviewer: ```suggestion
### Most Tests Accurately Reflected Purebred Dogs' Registered Breeds
```

For all dogs, the registered breed was always the majority estimate across tests (Figure @fig:pies; Supplementary Table 4).
However, most tests did not match every dog with its registered breed at 100%.
This primary outcome, selected *a priori* (Table @tbl:accuracy), therefore did not capture apparent qualitative differences in breed predictions (Figure @fig:pies).
In several cases, the breeds predicted by a test matched neither the DNA nor the photograph.
This pattern was particularly apparent for both the image classifier and one DTC genetic test (ViaPet by Accu-Metrics).
In the case of this genetic test, this pattern may have been influenced by the fact that the test predicted the registered breed zero times across both the control and treatment conditions (Table @tbl:accuracy).

Re-evaluating the results at the level of breed clades (Figure @fig:tern) suggested potential explanations for results that matched the breed of neither the DNA donor nor the photograph.

> In the cases of both NASNet and Accu-Metrics, the effects did not meet the level of statistical significance (Supplementary Table 5), but the odds ratios suggest that the condition of swapped photographs affected the predictions of both approaches.

Reviewer: ```suggestion
```

> The odds that these tests predicted a breed in the same clade as the DNA donor were 13 for NASNet and 11 for Accu-Metrics (Supplementary Table 5).

Reviewer: ```suggestion
The odds ratios for a breed in the same clade as the DNA donor were 13 for NASNet and 11 for Accu-Metrics, but these results are only suggestive as they did not meet our predefined threshold for significance (Supplementary Table 5).
```
CHECK THIS - ARE YOU SURE THIS IS DNA donor and not photograph? I may misunderstand this result.

Author: I think this is phrased to match the table below-- it's the odds ratio of the DNA donor's clade being in the results. I think I did it this way because it's very close to being the _a priori_ primary outcome we selected, just zoomed out to the clade level.

<img width="671" alt="Screenshot 2023-12-30 at 7 19 41 AM" src="https://github.com/rando2/dtcdog-ms/assets/8975841/ec508021-60d2-4771-a9f9-62f5d421415f">

```suggestion
When the tests were evaluated based on whether they predicted any breed in the same clade as the DNA donor at any percentage, NASNet performed with 100% accuracy in the control condition and 0% accuracy in the treatment condition, corresponding to an odds ratio of 13 (Supplementary Table 5).
Accu-Metrics identified breeds in the same clade as the donor twice in the control condition and zero times in the treatment condition, corresponding to an odds ratio of 11 (Supplementary Table 5).
However, these results are only suggestive, as neither met our predefined threshold for significance.
```
